### PR TITLE
ci: use drupalci webdriver image

### PR DESCRIPTION
**Pull request by millnut**
- Change the chrome webdriver image to align with the DrupalCI image (https://www.drupal.org/docs/develop/automated-testing/phpunit-in-drupal/running-phpunit-javascript-tests#docker-compose)
- Update the `MINK_DRIVER_ARGS_WEBDRIVER` to match DrupalCI
- Uses the same image as our workflow drupal-container

### CODE REVIEW

File: .lando.dist.yml [hunk 1]
```diff
@@ -23,10 +23,11 @@ services:
       environment:
         DRUSH_OPTIONS_ROOT: '/app/web'
         DRUSH_OPTIONS_URI: 'https://localgov.lndo.site'
+        SIMPLETEST_BASE_URL: 'http://localgov.lndo.site'
         SIMPLETEST_DB: 'mysql://database:database@database/database'
-        MINK_DRIVER_ARGS_WEBDRIVER: '["chrome", { "chromeOptions": { "w3c": false } }, "http://chromedriver:4444/wd/hub"]'
+        MINK_DRIVER_ARGS_WEBDRIVER: '["chrome", {"browserName": "chrome", "goog:chromeOptions": {"args": ["--disable-gpu","--headless", "--no-sandbox", "--disable-dev-shm-usage"]}}, "http://chromedriver:9515"]'
```
Comment (finnlewis) [inline .lando.dist.yml:28]: Out of interest, why the change of port to 9515 from 4444 ?

I was looking at this for reference: https://github.com/ddev/ddev-selenium-standalone-chrome/compare/main...FabbDev:ddev-selenium-standalone-chrome:fix-drupal-10-3

**Comment by finnlewis** [comment]
Let's merge this one, even though the tests are not running.

At least this allows us to run functional javascript tests locally in Drupal 10.3!

**Review by finnlewis — APPROVED** [review]

File: .lando.dist.yml (hunk 1 above)
Comment (millnut) [reply .lando.dist.yml:28]: Hi @finnlewis it's the default port used for the drupal-ci chromedriver image (see https://git.drupalcode.org/project/drupalci_environments/-/blob/dev/webdriver/chromedriver/Dockerfile?ref_type=heads#L57), the link you mentioned is for the ddev image which is a custom selenium-standalone-chrome

**Comment by stephen-cox** [comment]
This isn't working locally for me

```
An unexpected error occurred while starting Mink: unknown error: net::ERR_CONNECTION_REFUSED
  (Session info: chrome-headless-shell=123.0.6312.58)
  (Driver info: chromedriver=123.0.6312.58 (6b4b19e9dfbb93aa414dc045bd445287977d8d7a-refs/branch-heads/6312_46@{#3}),platform=Linux 6.5.0-41-generic x86_64)
```

Will need to investigate what's going wrong.

**Comment by millnut** [comment]
@stephen-cox is this ddev or lando?

**Comment by stephen-cox** [comment]
> @stephen-cox is this ddev or lando?

Lando

**Comment by millnut** [comment]
> > @stephen-cox is this ddev or lando?
> 
> Lando

Interesting have you run a `lando rebuild`, sometimes lando holds onto old config info unless a `lando rebuild` is run

**Comment by stephen-cox** [comment]
Yeah - I did a `lando destroy` as well as a rebuild and it's not working.

Just going through everything else prior to merge Tuesday so not done a full investigation yet. Will look further when I get a chance.

**Comment by millnut** [comment]
Thanks @stephen-cox I'll do some checks later

**Comment by finnlewis** [comment]
This doesn't work for @stephen-cox so needs a bit more testing.

**Comment by millnut** [comment]
Morning @stephen-cox I gave this a test this morning on an upgraded and clean install with Drupal 10.3 and it worked without issues:

```sh
lando phpunit web/modules/contrib/localgov_alert_banner/tests
PHPUnit 9.6.19 by Sebastian Bergmann and contributors.

Testing /app/web/modules/contrib/localgov_alert_banner/tests
..................                                                18 / 18 (100%)

Time: 02:25.887, Memory: 12.00 MB

OK (18 tests, 293 assertions)
```

Happy to jump on a call to look at this in some more detail

**Review by stephen-cox — APPROVED** [review]
I have tested this in an entirely new install and it works, so there's something going wrong with my local dev environments (I have a few).

Happy to approve.
